server/frontend/src/helpers.js

```diff
@@ -32,3 +32,17 @@ export const errorParser = (error) => {
   if (typeof message === "string") return message;
   return message.toString();
 };
+
+export const E_SERVER_ERROR = "Error while communicating with the server.";
+
+export const formatClientTimestamp = (datetime) => {
+  return new Intl.DateTimeFormat("en-US", {
```

Comment:
Can you just change `"en-US"` -> `undefined`?

Testing this in console I get:

| locale | result |
|---|---|
| `"en-US"` | `5/12/2021, 09:13 EDT` |
| `"en-CA"` | `2021-05-12, 09:13 EDT` |
| `"fr"` | `12/05/2021 à 09:13 UTC−4` |
| `"de"` | `12.5.2021, 09:14 GMT-4` |
| `undefined` | `2021-05-12, 09:13 EDT` (same as `"en-CA"`, for me) |